docs/guides/random-data-for-security.md

```diff
@@ -73,7 +73,7 @@ Likewise, since this is also a wrapper around the operating system's random data
 processes with the same parent returning the same data (i.e., it is "fork safe").
 
 If [Crypt::SysRandom::XS](https://metacpan.org/pod/Crypt::SysRandom::XS) is installed, it will use that to retrieve
-random bytes from system calls.  Note there may be issues when requesting more than 256 bytes on some systems when using the XS module.
+random bytes from system calls.
```

Comment:
Would it make sense to (somewhere) mention that we're assuming that the latest version of a given module is used, and that earlier versions may contain security issues?

Reply:
That seems unnecessary.